madmom/features/beats.py

```diff
@@ -390,6 +390,51 @@ def recursive(position):
     return np.array(positions)
 
 
+def detect_beats_with_ccf(activations, interval, look_aside=0.2):
```

Comment:
Yes, i think the same! At least i can use it for my thesis and evaluate it :)

I'm very curious now about the MultiAgent approach because, similar to `detect_beats` it always jumps from beat to beat but considers multiple tempo hypothesis and should be a very good fit for online mode 😋